addons/web_editor/models/field_html_history_mixin.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+from .patch_utils import apply_patch, generate_comparison, generate_patch
+
+
+class HtmlHistory(models.AbstractModel):
+    _name = "field.html.history.mixin"
+    _description = "Field html History"
+
+    history_revision_ids = fields.One2many(
+        "field.html.history.revision", "res_id", string="Related revision Ids",
```

Comment:
Missing a domain on `res_model`, currently the revisions are cross-models ^^

Reply:
Well yes. it is designed to be able to add history on any html field from any model in odoo.

Reply:
I don't understand what you mean. The revision are build to be able to be hooked on any html_field of any model.

Reply:
If you don't have a domain based on res_model, history_revision_ids contains revisions coming from article 3, task 3, note 3, ... res_id alone is not sufficient.

Reply:
Oh my bad, maybe with "Many2oneReference" it works magically. But I would add the domain to be explicit.

Reply:
I don't see how I could do that since I don't know which model I will be extending in the mixin.


Or I don't follow your idea at all, which is possible since I'm not an expert in python model at all.

Reply:
After IRL discussion, I've add a domain to `history_revision_ids` but I'm not even sure it is used by the ORM since it point to a Many2OneReference. 
Anyway, better safe than sorry I guess.